src/pmsearch/pmsearch_index.sh

```
@@ -0,0 +1,235 @@
+# for more details.
+#
+# Rebuild the pmsearch full-text search index from running PMCD
+# or a PCP archive.  Typically invoked nightly via systemd timer
+# or cron.
+#
+
+. $PCP_DIR/etc/pcp.env
+
+status=1
+tmp=`mktemp -d "$PCP_TMPFILE_DIR/pmsearch_index.XXXXXXXXX"` || exit 1
+trap "rm -rf $tmp \$NEW; exit \$status" 0 1 2 3 15
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)